### PR TITLE
Add README.md for Interpolation_Search #1324

### LINKED ISSUE #1324: Add README.md for Interpolation_Search
@marwahmanbir please assign this to me.

### PR DESCRIPTION
**Pull request by asha15**
@marwahmanbir 
Fixes #1324 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Create README.md
- 

#### Languages Used:
- 
- 

#### Files Added:
- README.md
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Interpolation_Search/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Pseudocode
+```
+A → Array list
+N → Size of A
+X → Target Value
+
+Procedure Interpolation_Search()
+
+   Set Lo  →  0
```
Comment (marwahmanbir) [inline Interpolation_Search/README.md]: Fix the indentation. 4 space, 8 space & so on...

Comment (asha15) [reply Interpolation_Search/README.md]: @marwahmanbir Done

**Comment by rishabhgarg25699** [comment]
@asha15 Please squash your commit also .

**Comment by asha15** [comment]
@rishabhgarg25699  I tried to squash by resetting all the commits,but it didn't work.
i'll appreciate if you could help me to do it

**Comment by marwahmanbir** [comment]
@asha15 you are left with squashing now. Could you try again squashing your commits? Do let us know if still you're facing issues.

**Comment by asha15** [comment]
@marwahmanbir  I tried this commands. But it doesn't seems to work
git reset origin/dev
git add --all
git commit -m 'my commit message'

**Pull request merged**